java/src/main/native/src/row_conversion.cu

```diff
@@ -937,7 +940,14 @@ __global__ void copy_strings_to_rows(size_type const num_rows, size_type const n
       }
       auto string_output_dest = &output_data[base_row_offset + offset];
       auto string_output_src = &variable_input_data[col][string_start_offset];
-      MEMCPY(string_output_dest, string_output_src, string_length, block_barrier);
+#ifdef ASYNC_MEMCPY_SUPPORTED
```

Comment:
One more question: Where is this macro defined? Will it be obsolete/outdated but we aren't notified?

Reply:
Ah I see that it is defined in this file :smile:

Reply:
This was added for Pascal support since Pascal doesn't support barriers, which are needed for `cuda::memcpy_async`. It would be difficult to have it fail as nvcc compiles for each architecture. I don't know how we would know if it did NOT compile a specific architecture.